Enable HSTS by default (#198)
humanmade/altis-security: docs/browser.md

The [`Strict-Transport-Security` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) (sometimes called HSTS) is used to enforce HTTPS (TLS/SSL) connections when loading a site and can be used to enhance the site's security.

> By default, Altis enables HSTS by default with the value `max-age=31536000; includeSubDomains`. You can configure the header using the `strict-transport-policy` setting:

Reviewer: Lose the second "by default".

Author: ```suggestion
By default, Altis enables HSTS with the value `max-age=31536000; includeSubDomains`. You can configure the header using the `strict-transport-policy` setting:
```

```json
{
	"browser": {
		"strict-transport-policy": "max-age=3600"
	}
}
```

> You can also switch the header off entirely by setting this to false:

Reviewer: "completely" not "entirely" as you have switched verb and from passive to active.

Author: ```suggestion
You can also switch the header off completely by setting this to false:
```

```json
{
	"browser": {
		"strict-transport-policy": false
	}
}
```

> Finally, if you set the value to `null` then Altis will send the header if the current request is using SSL.

Reviewer: "will send" or "will not send" when it is null?

Author: Will send, the browser security plugin reverts to a conditional check. Thinking I'll update the wording a bit though:
```suggestion
Finally, if you set the value to `null` then Altis _will_ send the header but only if the current request is already using HTTPS.
```
